src/klein/_session.py

```diff
@@ -21,10 +21,10 @@
 )
 
 if TYPE_CHECKING:               # pragma: no cover
+    from mypy_extensions import Arg, KwArg, VarArg
     from twisted.web.iweb import IRequest
     from twisted.python.components import Componentized
-    from mypy_extensions import KwArg, VarArg, Arg
-    from typing import TypeVar, Awaitable, Dict, Text
+    from typing import Awaitable, Dict, Sequence, Text, TypeVar
```

Comment:
The change here is importing Sequence, then re-ordering as if flake8-import-order checks this (it doesn't, though).